DOCS/man/options.rst

```diff
@@ -5555,15 +5555,15 @@ them.
         Disable any dithering done by mpv.
     auto
         Automatic selection.
-        On ``vo=gpu``: if output bit depth cannot be detected, 8 bpc is assumed.
-        On ``vo=gpu-next``: with ``gpu-api=d3d11``, real on-the-wire bpc is used.
-        For other ``gpu-api``, 8 bpc is used for SDR content.
+        On ``--vo=gpu``: detected depth or 8 bpc otherwise
+        On ``--vo=gpu-next``: detected depth or 8 bpc (for SDR target) or no
+        dithering otherwise.
     8
         Dither to 8 bit output.
 
     Note that the depth of the connected video display device cannot be
-    detected. Often, LCD panels will do dithering on their own, which conflicts
-    with this option and leads to ugly output.
+    detected except when using ``gpu-api=d3d11``. Explicitly setting your
+    display's bit depth is recommended.
```

Comment:
That's right, but applying dithering twice, is not as scary as this warning make it. It is better to dither to wire format either way. Even if there would be another dithering down the line, which we cannot really control/disable.

Reply:
> Why was the remark about LCD panel dithering deleted?

Because it was incorrect, there's no quality loss from dithering multiple times.

> It's possible for the wire format to be different from the LCD panel's capability

Changed the wording

Reply:
How about 
```
LCD panels may have a different actual depth compared to what they accept as input. In such situations, it's recommended to set the `dither-depth` parameter to match the panel's actual depth.
```
this value is indeed impossible to detect, even edid will not tell us true value.

Reply:
I can add that but I'd have thought `Explicitly setting your display's bit depth is recommended` was enough

Reply:
For me it is. Wait for @na-na-hi opinion.

Reply:
Hopefully this is good now